packages/is-shallow-equal/src/arrays.js

```diff
@@ -7,10 +7,6 @@
  * @return {boolean} Whether the two arrays are shallow equal.
  */
 export default function isShallowEqualArrays( a, b ) {
-	if ( a === b ) {
-		return true;
-	}
```

Comment:
Even though his path would never be reached after the `isShallowEqual` update, I think this would deoptimize standalone `isShallowEqualArrays` exports.